premium/usage.go

```diff
@@ -161,14 +153,14 @@ type BatchUpdater struct {
 	isClosed       bool
 }
 
-func NewUsageClient(pluginTeam cqapi.PluginTeam, pluginKind cqapi.PluginKind, pluginName cqapi.PluginName, ops ...UsageClientOptions) (*BatchUpdater, error) {
+func NewUsageClient(meta plugin.Meta, ops ...UsageClientOptions) (UsageClient, error) {
 	u := &BatchUpdater{
 		logger: zerolog.Nop(),
 		url:    defaultAPIURL,
 
-		pluginTeam: pluginTeam,
-		pluginKind: pluginKind,
-		pluginName: pluginName,
+		pluginTeam: meta.Team,
```

Comment:
just use meta instead?

Reply:
Done in [`27b0d28` (#1436)](https://github.com/cloudquery/plugin-sdk/pull/1436/commits/27b0d28d55e054d001f1e1332d150f5f6fec8e39)